.github/workflows/build-images.yaml

```diff
@@ -0,0 +1,87 @@
+name: Build core and operator images
```

Comment:
```suggestion
name: Paladin OCI
```
Or `Paladin / OCI / Build` ?

Think we should come up with a consistent naming (and casing!) scheme for the workflows so they are clear how they are organized, what each accomplishes, etc.

Here's what it looks like currently:

<img width="343" alt="Screenshot 2024-11-05 at 18 21 39" src="https://github.com/user-attachments/assets/90a8a320-af07-473a-ab86-433d93de20a0">

Reply:
Okay I propose the current workflows list should read as follows:
* `Housekeeping`
* `Docs`
* `Helm / Release`
* `CI / Build
* `Operator / Build`
* `PR / Build` --> how is this different from CI ?
* `OCI / Release` (i.e. release-images.yaml)
* `OCI / Latest` (i.e. trigger-images.yaml)

Reply:
I updated the PR description, this is not final yet, and I would like to use the built images for the operator CI, but that is for the next iteration